lib/govuk_app_config/govuk_error/configuration.rb

```diff
@@ -3,20 +3,82 @@
 
 module GovukError
   class Configuration < SimpleDelegator
-    attr_reader :data_sync, :sentry_environment
-    attr_accessor :active_sentry_environments, :data_sync_excluded_exceptions
+    attr_reader :data_sync
+    attr_accessor :data_sync_excluded_exceptions
 
     def initialize(_sentry_configuration)
       super
-      @sentry_environment = ENV["SENTRY_CURRENT_ENV"]
       @data_sync = GovukDataSync.new(ENV["GOVUK_DATA_SYNC_PERIOD"])
-      self.active_sentry_environments = []
-      self.data_sync_excluded_exceptions = []
+      set_up_defaults
+    end
+
+    def set_up_defaults
```

Comment:
It occured to me after I wrote the experimental code that this method could belong in `govuk_error.rb`, so we'd have something like this:

```rb
config = Configuration.new(sentry_config)
set_up_defaults(config)
yield config if block_given?
```

I think it's slightly nicer because it keeps the `Configuration` class responsible for just wrapping the Sentry class, but to be honest, I don't really mind either way.

Reply:
I think having it in `Configuration` is slightly nicer now, to be honest. We can only make one call to `GovukError.configure`, which under the hood is now responsible for merging in custom `before_send` logic, including datasync-error-ignoring, etc. I think it's quite nice that there's just one file to look at all of the default configuration (including the logic above), so there's a little bit less poking around involved if you want to override something in your `GovukError.configure` call.